### ISSUE TITLE
Editor 2 error

### ISSUE THREAD

**Issue by AKashpur** [defold/editor2-issues#1676]
Editor 2 crashes when i attempt to bundle debug version for android
### Expected behaviour
      generate bundle
### Actual behaviour
       crash
### Steps to reproduce

<hr/>

<table>
<tr><td>Build time</td><td>2018-02-19T13:59:00.370637</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>2f83d73448eca2d13be5bcd21a14a1c5ce03d07b</td></tr>
<tr><td>Defold engine sha</td><td>9988355914f63362a4292c4827639f0187275e70</td></tr>
<tr><td>Defold version</td><td>1.2.121</td></tr>
<tr><td>Error</td><td><a href='https://sentry.io/defold/editor2/?query=id%3A"2c85a8a08a8445ec9c0b055d5bbec83e"'>2c85a8a08a8445ec9c0b055d5bbec83e</a></td></tr>
<tr><td>GPU</td><td>GeForce GTS 450/PCIe/SSE2</td></tr>
<tr><td>GPU Driver</td><td>4.5.0 NVIDIA 375.70</td></tr>
<tr><td>Java version</td><td>1.8.0_102-b14</td></tr>
<tr><td>OS arch</td><td>amd64</td></tr>
<tr><td>OS name</td><td>Windows 7</td></tr>
<tr><td>OS version</td><td>6.1</td></tr>
<table>

**Comment by AKashpur** [comment]
the problem was with fetching library

**Comment by erikangelin-king** [comment]
Reopening, because you should get a nicer error for this "normal" error case.

**Comment by erikangelin-king** [comment]
@AKashpur What (broken) dependency did you add to provoke the error?

**Comment by AKashpur** [comment]
zip archieve of library from github. is seems like editor cached broken version of zip archieve and error happened when it tried to unpuck it. when i deleted .internal and fetched lib again, editor created bundle as expected

**Comment by AKashpur** [comment]
yeah, sorry for wasting your time, it was indeed error, not crash, i'm exhausted and used wrong term

**Comment by andsveking** [comment]
We need to handle corrupt library zip better. Added an issue for this: **DEF-3151**

**Comment by AGulev** [comment]
related? https://github.com/defold/editor2-issues/issues/1316
